node/src/mesh/mod.rs

```diff
@@ -1,8 +1,25 @@
+use mpc_keys::hpke;
+
 use crate::protocol::contract::primitives::Participants;
 use crate::protocol::ProtocolState;
 
 pub mod connection;
 
+#[derive(Clone, Debug)]
+pub struct NetworkConfig {
+    pub sign_sk: near_crypto::SecretKey,
+    pub cipher_pk: hpke::PublicKey,
+}
+
+impl NetworkConfig {
+    pub fn test() -> Self {
```

Comment:
Does this mean that all the nodes in our tests will have the same keys? What if we break our contract or node logic and mix all the keys? Tests will not catch that, but our real env will be broken after the release.

Reply:
this is actually unused code -- will delete the function. The tests use different keys per node so nothing to worry about